README.md

```diff
@@ -65,9 +65,13 @@ Only `linux-amd64-libcxx-libinstana_sensor.so` is required.
 
 ## Release History
 
+### 1.9.1 (2024-08-22)
+
+  * [Minor] Add a script to build Envoy libinstana sensor locally.
```

Comment:
This is irrelevant for customers as our code is not OpenSource.

Just write `No changes regarding Envoy Proxy tracing`.